samza-core/src/test/java/org/apache/samza/task/TestAsyncRunLoop.java

```diff
@@ -86,7 +86,7 @@ TaskInstance createTaskInstance(AsyncStreamTask task, TaskName taskName, SystemS
     scala.collection.immutable.Set<SystemStreamPartition> sspSet = JavaConverters.asScalaSetConverter(Collections.singleton(ssp)).asScala().toSet();
     return new TaskInstance(task, taskName, mock(Config.class), taskInstanceMetrics,
         null, consumers, mock(TaskInstanceCollector.class), mock(SamzaContainerContext.class),
-        manager, null, null, sspSet, new TaskInstanceExceptionHandler(taskInstanceMetrics, new scala.collection.immutable.HashSet<String>()), null, null);
+        manager, null, null, null, sspSet, new TaskInstanceExceptionHandler(taskInstanceMetrics, new scala.collection.immutable.HashSet<String>()), null, null);
```

Comment:
minor: if the null input to TableManager is only from unit tests, it seems that it can be easily solved by using a mock in the test. Your call.

Reply:
Same as above.